File: packages/core/test/fixtures/regex/objc/launchdarkly/positive/basic.m
#import <LaunchDarkly/LDClient.h>

@implementation Checkout
- (NSString *)run {
    BOOL enabled = [[LDClient get] boolVariation:@"CHECKOUT_V2" defaultValue:NO];
    if (enabled) {
        return @"v2";
    }
    return @"v1";
}
@end

File: packages/core/test/fixtures/regex/swift/launchdarkly/positive/basic.swift
import LaunchDarkly

class Checkout {
    let client: LDClient

    init(client: LDClient) {
        self.client = client
    }

    func run() -> String {
        let enabled = client.boolVariation(forKey: "CHECKOUT_V2", defaultValue: false)
        if enabled {
            return "v2"
        }
        return "v1"
    }
}
